src/util/address.rs

```diff
@@ -145,6 +154,173 @@ impl FromStr for AddressType {
     }
 }
 
+/// Version of the WitnessProgram: first byte of `scriptPubkey` in
+/// transaction output for transactions starting with opcodes ranging from 0
+/// to 16 (inclusive).
+///
+/// Structure helps to limit possible version of the witness according to the
+/// specification; if a plain `u8` type will be used instead it will mean that
+/// version > 16, which is incorrect.
```

Comment:
Did you mean to say "if a plain `u8` type was used instead it would mean that version may be > 16, which is incorrect"?